src/theme/components/global.ts

```diff
@@ -4,13 +4,12 @@ export { wrapperStyles } from "./globalMixins";
 /** Reusable component styles. */
 
 // Used in `Select` and `TextInput`.
-const activeFocus = {
-  border: "1px solid",
-  borderColor: "ui.focus",
+const activeFocus = (darkMode = false) => ({
```

Comment:
Prepping for what we discussed this morning?

Reply:
Actually, this is based on the current SCSS `focus-outline` mixin. I didn't want to deviate from what was existing, just move it over from scss to css-in-js.